test/XRP/issued-currency-client.test.ts

```diff
@@ -619,6 +619,64 @@ describe('Issued Currency Client', function (): void {
     })
   })
 
+  it('getTransferFee - successful response', async function (): Promise<void> {
+    // GIVEN an IssuedCurrencyClient with mocked networking that will return a successful hash for submitTransaction
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingGrpcClient,
+      fakeSucceedingWebSocketClient,
+      XrplNetwork.Test,
+    )
+
+    const expectedTransferFee = 1000000012
+
+    // WHEN setTransferFee is called
+    const transferFee = await issuedCurrencyClient.getTransferFee(
+      this.wallet.getAddress(),
+    )
+
+    // THEN a transfer rate and is the expected value
```

Comment:
```suggestion
    // THEN the expected value is returned.
```
(or something like this)